omero/analyze/index.html

```diff
@@ -32,7 +32,7 @@ <h2 class="header-blue">Analyze your data in OMERO</h2>
                 <p>Draw ROIs across multiple dimensions and measure pixel
                     intensity. </p>
                 <a class="button hollow tiny" href="http://help.openmicroscopy.org/measurement-tool.html" target="_blank">View Insight user guide</a>
-                <a class="button hollow tiny" href="https://www.youtube.com/watch?v=FQD540TTwFg" target="_blank">Watch OMERO.iviewer movie</a>
+                <a class="button hollow tiny" href="https://www.youtube.com/watch?v=FQD540TTwFg" target="_blank">Watch OMERO.<span style="text-transform: lowercase;">i</span>viewer movie</a>
```

Comment:
It will still be "verbose" though because this is a one-off that overrides a style applied to all the buttons.